pkg/ccl/backupccl/manifest_handling.go

```diff
@@ -436,17 +441,33 @@ func writeBackupManifest(
 		}
 	}
 
-	if err := cloud.WriteFile(ctx, exportStore, filename, bytes.NewReader(descBuf)); err != nil {
-		return err
+	// The backup manifest should only be written to the base directory,
+	// unlike the checkpoints which can be written to both the base and
+	// progress directory depending on the version.
+	if filename != backupManifestName {
+		if err := writeNewCheckpointFile(ctx, settings, exportStore, filename, descBuf); err != nil {
+			return err
+		}
+	} else {
+		if err := cloud.WriteFile(ctx, exportStore, filename, bytes.NewReader(descBuf)); err != nil {
+			return err
+		}
 	}
 
 	// Write the checksum file after we've successfully wrote the manifest.
 	checksum, err := getChecksum(descBuf)
 	if err != nil {
 		return errors.Wrap(err, "calculating checksum")
 	}
-	if err := cloud.WriteFile(ctx, exportStore, filename+backupManifestChecksumSuffix, bytes.NewReader(checksum)); err != nil {
-		return errors.Wrap(err, "writing manifest checksum")
+
+	if filename != backupManifestName {
```

Comment:
really wish this and the read path were better structured so actual manifest reading/writing didn't share anything with checkpoint manifest reading and writing. Maybe add a TODO for a future cleanup, feel free to put my name on it.